front-end-app/src/components/ModalSimpleForm.js

```diff
@@ -9,34 +9,27 @@ const inlineStyle = {
     marginRight: 'centered'
   }
 };
-class ModalSimpleForm extends React.Component {
-
-
-  render () {
-    const form = this.props.form;
-
+const ModalSimpleForm = (props) =>{
```

Comment:
Certainly not required but I like to destructure right from the argument in functional components. I think it helps describe the component in a similar way that the `PropTypes` do:
```suggestion
const ModalSimpleForm = ({ isOpen, title, form, modalClose }) =>{
```

Reply:
Yup, I have seen this elsewhere. Agreed, much easier to understand the component's function from the onset.